H/HDF5/build_tarballs.jl

```diff
@@ -28,6 +28,9 @@ if [[ ${target} == x86_64-*mingw* ]]; then
     mv mingw64/bin/*.dll ${prefix}/bin
 elif [[ ${target} == i686-*mingw* ]]; then
     mv mingw32/bin/*.dll ${prefix}/bin
```

Comment:
shouldn't these all be moved to the `{prefix}/lib` folder (another PR)?

Reply:
I think that dlls goes to `${prefix}/bin` on Windows

Reply:
ah ok